admin-dev/themes/default/template/header.tpl

```diff
@@ -308,8 +308,9 @@
               {if isset($maintenance_mode) && $maintenance_mode == true}
                 <span class="shop-state" id="maintenance-mode">
                   <i class="material-icons">build</i>
-                  <span class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
-                    title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in maintenance.'}</strong></p><p class='text-left'>{l s='Your visitors and customers cannot access your shop while in maintenance mode.%s To manage the maintenance settings, go to Preferences > Maintenance.' sprintf='<br />'}</p>">{l s='Maintenance mode'}</span>
+                    <a class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
```

Comment:
paragraph in span is also not a good idea but this paragraph is only for a tooltip, right?